modules/vendor.js

```diff
@@ -1,12 +1,16 @@
 'use strict';
 
-const events = require('../events');
 const faker = require('faker');
 require('dotenv').config();
 
+const io = require('socket.io-client');
+
+const capsConnection = io.connect(`${process.env.HOST}/caps`);
```

Comment:
Same thing as the driver file, but also do it to the ` STORE_NAME` and `VENDOR_ONE`